rest/taskana-rest-spring-example-boot/src/main/java/pro/taskana/rest/ExampleRestApplication.java

```diff
@@ -1,49 +1,30 @@
 package pro.taskana.rest;
 
-import javax.annotation.PostConstruct;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.context.annotation.ComponentScan;
-import org.springframework.context.annotation.Import;
 import org.springframework.scheduling.annotation.EnableScheduling;
 
-import pro.taskana.RestConfiguration;
-import pro.taskana.jobs.TransactionalJobsConfiguration;
 import pro.taskana.sampledata.SampleDataGenerator;
 
-/** Example Application showing the implementation of taskana-rest-spring. */
-@SpringBootApplication
+/** Example Application showing a minimal implementation of the taskana REST service. */
 @EnableScheduling
-@ComponentScan(basePackages = "pro.taskana")
-@SuppressWarnings("checkstyle:Indentation")
-@Import({
-  ExampleRestConfiguration.class,
-  TransactionalJobsConfiguration.class,
-  RestConfiguration.class,
-  WebMvcConfig.class
-})
+@EnableAutoConfiguration
+@ComponentScan("pro.taskana")
 public class ExampleRestApplication {
 
-  private final SampleDataGenerator sampleDataGenerator;
-
-  @Value("${generateSampleData:true}")
-  public boolean generateSampleData;
-
   @Autowired
```

Comment:
Just fyi: As of Spring 4.3, you no longer need to specify an explicit injection annotation in a single-constructor scenario. (meaning the @Autowired could be deleted here)